ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,49 @@
+## Description
+
+Provide a description of your issue.
+For any suggestions, please address them on the [forum](https://forum.freegamedev.net/viewforum.php?f=16).
+
+## Steps to reproduce
+
+## Configuration
+
+STK release version:
+
+STK Source (ppa, distribution package, official bin,...):
+If you did not download STK from the official website, please specify where you got it:
+
+System: 
+
+Graphics card: 
+
+CPU: 
+
+Gamepads/keyboards models if related to the issue:
+
+## Additional informations
```

Comment:
Please correct to "Additional information" instead of "Additional informations"

Reply:
Fixed, thanks :) Those always singular nouns are hard to remember when coming from french.